src/jinacld_tools/aws/services/s3.py

```diff
@@ -0,0 +1,37 @@
+import glob
+from pathlib import Path
+
+from jinacld_tools.aws.logger import get_logger
+from jinacld_tools.aws.client import AWSClientWrapper
+
+
+class S3:
+    """Wrapper around boto3 to upload to/download from S3 bucket
+    """
+
+    def __init__(self, bucket: str):
+        self._client = AWSClientWrapper(service='s3').client
+        self._bucket = bucket
+        self._logger = get_logger(self.__class__.__name__)
+
+    def add(self, path: str, key: str):
+        if not Path(path).exists():
+            self._logger.error(f'Invalid path: {path}! Nothing to upload!')
+            return
```

Comment:
Maybe its better to raise an exception here?

Reply:
Agree. https://github.com/jina-ai/cloud-helper/pull/3/commits/408ba7cbfda83f02197bda7e8bb6f92be7f72dd1